docs/miscellaneous.rst

```diff
@@ -322,43 +321,39 @@ The following is the order of precedence for operators, listed in order of evalu
 Global Variables
 ================
```

Comment:
This section has its own bigger section `units-and-global-variables`. Do we want to keep both?

Reply:
I think it's good to document them in their own section and also here as part of the cheat sheet.